### ISSUE TITLE
PIFFS: mount failed, -10025 Error

### ISSUE THREAD

**Issue by sw-dev-code** [fabianoriccardi/esp-logger#8]
When I run the example code in platformiIO I'm getting the following error:

```
E (57) SPIFFS: mount failed, -10025
[E][SPIFFS.cpp:72] begin(): Mounting SPIFFS failed! Error: -1
[E][vfs_api.cpp:22] open(): File system is not mounted
```

How can I solve this issue? I've erased the flash before testing the example code.

**Comment by fabianoriccardi** [comment]
Hi, had you tried to run the ESP32 default examples about SPIFFS?

**Comment by sw-dev-code** [comment]
@fabiuz7 Yes, the only thing that solved my issue is to set the `  if(!ESP_LOGGER_FLASH_FS.begin(true)){` in logger-spiffs.cpp. Is there some step that I'm missing before flashing the firmware?

**Comment by fabianoriccardi** [comment]
Hi, did you finally solved the issue?
